strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BaseController.java

```diff
@@ -44,7 +49,12 @@
 
     protected Configuration getConfiguration()
     {
-        return configurationManager.getConfiguration();
+        return configurationManagementService.getConfiguration();
+    }
+
+    protected MutableConfiguration getMutableConfigurationClone()
```

Comment:
@fuss86 do we really need both `Configurations` here?

Reply:
I wanted to avoid it but at some point it turned out it's impoosible. I'll double check later, why

Reply:
mutable representations are used in the Controllers (as response body, for example, since they have properly configured JAXB/JSON annotations) and tests. 

We could get rid of it in the follow up task, I hope. Refactoring current behavior is not complicated, but may be time consuming.